handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -600,6 +602,14 @@ public SslContextBuilder enableOcsp(boolean enableOcsp) {
         return this;
     }
 
+    /**
+     * @param secureRandom the source of randomness for SslContext
```

Comment:
```suggestion
    /**
     * Specify a non-default source of randomness for the {@link SSLContext}.
     * <p>
     * In general, the best practice is to leave this unspecified, or to assign a new random source using the
     * default {@code new SecureRandom()} constructor.
     * Only assign this something when you have a good reason to.
     *
     * @param secureRandom the source of randomness for {@link SSLContext}
```

Reply:
I think this explanation is very good.

My understanding is that this is only used in the JDK SslProvider (JdkSslServerContext JdkSslClientContext), is that correct? If so, what do you think about adding this to the comments?

Reply:
@thxwelchs please add this.

Reply:
Added!
4408ced

https://github.com/netty/netty/blob/047d95aa08fdd8fed84835a9544b792e74a24025/handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java#L606-L612